tests/e2e/suites/registration.test.ts

```diff
@@ -0,0 +1,31 @@
+import { setupAndGo } from '../utilities/setup-and-go';
+
+describe('Test Basic login flow', () => {
+  ['chromium', 'webkit', 'firefox'].forEach((browserType) => {
+    it(`should login successfully and then log out with ${browserType}`, async (done) => {
```

Comment:
... similar to the above... "should register successfully..."

Reply:
Fixed.